components/common/IconSwitch.tsx

```diff
@@ -8,7 +8,8 @@ import { Switch as HeadlessSwitch } from '@headlessui/react'
 
 export interface IconSwitchProps extends ComponentProps<'div'> {
   on?: boolean
-  icon: ReactNode
+  icon?: ReactNode
```

Comment:
Parameter names are not very clear. Let's rename `icon -> offIcon` and `iconActive -> onIcon`